package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-reloaded",
-  "version": "0.1.3",
+  "version": "1.0.0",
```

Comment:
This version bump prevents NPM from trying to automatically install the new version (which has breaking changes) in downstream projects (e.g. ginger-content-gatsby)